pkg/testutils/mockmaps/lbmap.go

```diff
@@ -16,29 +16,37 @@ import (
 
 type LBMockMap struct {
 	lock.Mutex
-	BackendByID            map[lb.BackendID]*lb.Backend
-	ServiceByID            map[uint16]*lb.SVC
-	AffinityMatch          datapathTypes.BackendIDByServiceIDSet
-	SourceRanges           datapathTypes.SourceRangeSetByServiceID
-	DummyMaglevTable       map[uint16]int // svcID => backends count
+	BackendByID   map[lb.BackendID]*lb.Backend
+	ServiceByID   map[uint16]*lb.SVC
+	AffinityMatch datapathTypes.BackendIDByServiceIDSet
+	SourceRanges  datapathTypes.SourceRangeSetByServiceID
+	// metrics for upsertMaglevLookupTable mock
+	UpsertMaglevLookupTableCallCnt map[uint16]int
```

Comment:
It looks like this metric is currently not checked in any test. Should it be?

Reply:
Probably not needed. See the below comment around `UpsertServiceCallCnt`.